code/game/mecha/mech_fabricator.dm

```diff
@@ -10,9 +10,6 @@
 	active_power_usage = 5000
 	req_access = list(access_robotics)
 	var/time_coeff = 1
-	var/resource_coeff = 1
-	var/time_coeff_tech = 1

```

Comment:
Sure thing.

So mech fabs have a special thing where increasing the materials levels decreases the cost of designs.

They also have a thing where increasing the programming level makes things produced faster as well.

I removed both; I didn't remove faster production by upgrading the *parts* only by increasing the programming level.

It didn't make much sense to have speed increased by two independent factors; figured it best for it to be handled *purely* by part upgrades.